.github/workflows/main.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         shell: bash
         run: |
           curl -fsS -o get-poetry.py https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
-          python get-poetry.py -y
+          python get-poetry.py -y --preview
```

Comment:
We had reverted this earlier right?

Reply:
Yes, but with the change of lock file format in #2361 we need to use the preview version for now. This will be reverted back once the stable version of `1.1.0` is released.